apps/charterafrica/src/lib/ecosystem/payload.js

```diff
@@ -0,0 +1,91 @@
+import * as Sentry from "@sentry/nextjs";
+
+import { localizeData } from "@/charterafrica/lib/ecosystem/utils";
+import api from "@/charterafrica/lib/payload";
+
+export async function updateOrCreate(collection, toCreate, locale) {
+  try {
+    const { docs } = await api.getCollection(collection, {
+      locale,
+      where: {
+        airtableId: { equals: toCreate.airtableId },
+      },
+    });
+    if (docs.length) {
+      const data = await api.updateCollection(
+        collection,
+        docs[0]?.id,
+        {
+          ...toCreate,
+          updatedAt: new Date(),
```

Comment:
My understanding Collections automatically supports `createdAt` and `updatedAt` unless specifically disabled via [`timestamps`](https://payloadcms.com/docs/configuration/collections#options) option. Why are we manually updating `updatedAt`?